### DISCUSSION TITLE
QUESTION on adding videos

### DISCUSSION THREAD

**Discussion by adamesq** [rrirower/Channel-Feed-Manager#99] [Q&A] [answered]
If you put more than one video in an "Videos in this content", for example, when adding a short form video, will the 2 videos play back-to-back? 

I'm thinking I could put a 3 second "Station ID" video in here to play before the 2nd, main video. Is that right?
![question](https://github.com/rrirower/Channel-Feed-Manager/assets/34196956/558c20d9-189f-4658-b01b-35d234382082)

**Comment by rrirower** [comment, marked as answer]
No that's incorrect. The Roku json spec allows for specifying different resolutions of video content.  For example, HD or SD. This is typically done to support older Roku devices. Although, I don't see too many channel developers adding anything but HD content.

For more info, reference the Roku json spec.
